docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md

```diff
@@ -81,6 +81,40 @@ helm repo add prometheus-community https://prometheus-community.github.io/helm-c
 helm install -n kube-system --version 2.13.2 kube-state-metrics prometheus-community/kube-state-metrics
 ```
 
+## Storage class considerations
+
+The type of volume provisioned for YugabyteDB Anywhere as well as YugabyteDB depends on the Kubernetes storage class being used. Consider following points when choosing or creating a storage class:
+
+1. It is recommended to set volume binding mode on a storage class to `WaitForFirstConsumer` for dynamically provisioned volumes. This will delay provisioning until a pod using the persistent volume claim (PVC) is created. The pod topology or scheduling constraints will be respected.
+
+   However, scheduling might fail if the storage volume is not accessible from all the nodes in a cluster and the default volume binding mode is set to `Immediate` for certain regional cloud deployments. The volume may get created in a location/zone that is not accessible to the pod causing the failure.
+
+   For more information, see [Kubernetes: volume binding mode](https://kubernetes.io/docs/concepts/storage/storage-classes/#volume-binding-mode).
+
+   On Google Cloud Provider (GCP), if you choose not to set binding mode to `WaitForFirstConsumer`, you might use regional persistent disks to replicate data between two zones in the same region on Google Kubernetes Engine (GKE). This can be used by the pod, in cases when the pod reschedules to another node in a different zone. For more information, see the following:
```

Comment:
This can be used by the pod, in cases when the pod reschedules to another node in a different zone.  >  This can be used by the pod when it reschedules to another node in a different zone.